app/script/media/MediaStreamHandler.js

```diff
@@ -663,6 +663,9 @@ z.media.MediaStreamHandler = class MediaStreamHandler {
    * @returns {undefined} No return value
    */
   _showPermissionDeniedHint(mediaType) {
+    if (mediaType === z.media.MediaType.AUDIO_VIDEO || mediaType === z.media.MediaType.VIDEO) {
+      this.mediaRepository.showNoCameraModal();
```

Comment:
Shouldn't you return here? You are going to show 2 modals in that case, is it normal?

Reply:
Nope, the other is just a log.

Reply:
Of course, it would have also shown the warning banner.

Reply:
Humm not sure I follow. The following doesn't seem to be a log, it sends a `z.event.WebApp.WARNING.MODAL` event, which let me think that it's going to show a modal